### PR TITLE
Jk/merge/master feature rds at b3263aff4

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Merge master @b3263aff4 to feature-rds branch

Addresses [CUMULUS-1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

-- master -> feature RDS PR merge at b3263aff4.  No major conflicts/callouts other than I deliberately left the duplicate "Added" key in the CL for unreleased changes as it will make future merges easier *until* release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,9 +18,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - The `async-operations` endpoint will now omit `output` instead
       of returning `none` when the operation did not return output.
   - **CUMULUS-2309**
-    - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor of `@cumulus/api/lib/granule-remove-from-cmr.unpublishGranule` and `@cumulus/api/lib/granule-delete.deleteGranuleAndFiles`.
+    - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor
```
Comment (Jkovarik) [inline CHANGELOG.md:21]: Changes here were almost entirely the result of an editor re-wrap on accepting changes.  I can revert if that's obnoxious, however we *did* agree it was good practice at one point.

Comment (Jkovarik) [reply CHANGELOG.md:21]: Test comment

**Comment by nemreid** [comment]
This is what I see as conflicts:
image
Going to review these files in particular and skim the rest.

**Comment by Jkovarik** [comment]
@nemreid  thanks, it probably would have been helpful to post that.... sorry! 😓

File: CHANGELOG.md (hunk 1 above)
Comment (nemreid) [reply CHANGELOG.md:21]: Isn't it going to result in a conflict when merging back to master?

Comment (Jkovarik) [reply CHANGELOG.md:21]: These are all in the RDS feature list, so.... I don't think so?

Comment (nemreid) [reply CHANGELOG.md:21]: _expands_, Oh. Fair enough!

**Review by nemreid — CHANGES_REQUESTED** [review]
I'm seeing a lot of `package.json`s that don't exist on the main branch that still have `7.1.0` as the version number for themselves and dependencies. Those should be updated. Also, before I forget to mention it later: quick reminder that this merge must *not* be squashed.

**Comment by Jkovarik** [comment]
@nemreid  right you are, good catch.  6ffe1a182

**Comment by Jkovarik** [comment]
Let's try that again:  6ffe1a182

**Review by nemreid — APPROVED** [review]
LGTM 👍 Remember not to squash.. 🙂

**Pull request merged**